src/lightkurve/targetpixelfile.py

```diff
@@ -1146,6 +1147,62 @@ def plot(
                         ax.add_patch(rect)
         return ax
 
+    def _to_matplotlib_animation(self, step=None, interval=200, **plot_args):
```

Comment:
If you are hard coding a default later (`step = 50` perhaps the default should be set up here rather than `None`, `None` steps never makes sense.